packages/core/src/grid/docs/async-data.stories.ts

```diff
@@ -39,6 +43,7 @@ export function asyncData() {
     private async load() {
       this.grid = { label: '', rowActions: [], columns: [], rows: [] } as unknown as DemoGrid;
       this.grid = (await DemoService.asyncData).grid;
+      this.gridElement.requestUpdate(); // <- we need this so the i18n hydrates itself after load
```

Comment:
The async functionality was rendering the i18n strings before it had values to put in them. Forcing an update here helped solve that. Leaving in as a good practice/example.